style.css

```diff
@@ -0,0 +1,671 @@
+* {
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+  text-decoration: none;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  background-color: #e5e5e5;
+}
+
+nav {
+  background-color: #fff;
+  border-radius: 5px;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 0 0 50px;
+}
+
+.logo {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.navbar-links ul {
+  display: flex;
+}
+
+.navbar-links li a {
+  padding: 1rem;
+  display: block;
+}
+
+.menu-button {
+  position: absolute;
+  right: 1rem;
+  display: none;
+}
+
+@media (max-width: 500px) {
+  .menu-button {
+    display: flex;
```

Comment:
- [ ] **[OPTIONAL]:** Great job using media queries! 💯 However, I would suggest that you use `min-width` instead of `max-width` since you're using a mobile-first approach. The `min-width` instruction means that when the screen has a minimum width of `Xpx`, apply the styles inside the `{ }` brackets 🤓, giving you the possibility to define styles from `Xpx` to pretty much infinite or until the next breakpoint, if there's one.  Another suggestion would be to change your breakpoint value to `768px` instead of `500px`, this is to make sure the devices you're targeting are Desktop devices.

I'm sorry that the previous reviewer didn't point out this, but it would be great if you fix it! 🤓 🛠️